library/core/src/cell.rs

```diff
@@ -185,6 +184,12 @@
 //! }
 //! ```
 //!
+//! [`Arc<T>`]: ../../std/sync/struct.Arc.html
+//! [`Rc<T>`]: ../../std/rc/struct.Rc.html
+//! [`RwLock<T>`]: ../../std/sync/struct.RwLock.html
+//! [`Mutex<T>`]: ../../std/sync/struct.Mutex.html
+//! [`atomic`]: ../../core/sync/atomic/index.html
```

Comment:
```suggestion
//! [`Arc<T>`]: crate::sync::Arc
//! [`Rc<T>`]: crate::rc::Rc
//! [`RwLock<T>`]: crate::sync::RwLock
//! [`Mutex<T>`]: crate::sync::Mutex
//! [`atomic`]: crate::sync::atomic
```
Intra-doc-links should be used now.

Reply:
I don't think that is possible.  These docs live in `core`, but most of those only exist in `std` or `alloc`.  I do not believe intra-doc-links support that.